src/views/authenticated/List.tsx

```diff
@@ -25,9 +26,7 @@ export function List({ data: unfilteredListItems, listPath }: Props) {
 	if (unfilteredListItems.length === 0) {
 		return (
 			<>
-				<p>
-					Hello from the <code>/list</code> page!
-				</p>
+				<h1>{listName}</h1>
```

Comment:
nitpick (non-blocking): we may want to change the list name to `h2` since the header name `Smart Shopping List` is already `h1` and I guess I just had this idea that you are noly suppose to use 1 per page but as I type this I am not sure if that is a real "rule" or something I've just thought this whole time 😅🤔😂

Reply:
I dont think thats a nitpick, I think thats valid! You mentioning this actually had me thinking about the placement of our app title "smart shopping list". We can talk about this more when we all discuss design work, but having the app name in place of the "home view" as the nav bar could be a good idea. Then the name of the list could be the central figure in the page view and take on the <h1/> tag?!